qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -91,3 +91,8 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
+            event_check = "GUEST_PANICKED"
+            event_cmd = service kdump stop;echo c >/proc/sysrq-trigger
```

Comment:
I think it should be`service kdump stop && echo c >/proc/sysrq-trigger`
If previous command failed with `;` the second one will run, which is not what we want. 
But with `&&` the second one will not run.

Reply:
@hereischen  updated and please help to review it? thanks

 (17/17) smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_guest_panicked.from_guest: PASS (107.23 s)